features/bevragen/overzicht.md

```diff
@@ -0,0 +1,80 @@
+
+Is de burgerservicenummer van de te raadplegen persoon/personen niet bekend, dan kan deze worden opgezocht met behulp van de [Zoek persoon](#zoeken-van-personen) operaties.
+
+## Wet bescherming persoonsgegevens
+
+De BRP bevragen API is ontworpen conform de REST principes. Om ook aan de Wet bescherming persoonsgegevens te conformeren zijn er concessies gedaan met betrekking tot het toepassen van de REST principes. De belangrijkste concessie is dat de POST methode en niet de GET methode wordt gebruikt om personen te bevragen. Dit zorgt er voor dat er geen [persoonlijk identificeerbare informatie (PII)](https://piwikpro.nl/blog/pii-niet-pii-en-persoonsgegevens/) terecht komen in de url van een request en daardoor ook niet in server logs.
+
+## Zoeken van personen
+
+De volgende zoek operaties kunnen worden gebruikt om een persoon met niet-uniek identificerende persoonsgegevens te vinden:
+
+- zoek met geslachtsnaam en geboortedatum
+  - [overzicht](./zoek-met-geslachtsnaam-en-geboortedatum/overzicht.feature)
+  - [fout cases](./zoek-met-geslachtsnaam-en-geboortedatum/fout-cases.feature)
+- zoek met geslachtsnaam, voornamen en gemeente van inschrijving
+  - [overzicht](./zoek-met-geslachtsnaam-voornamen-en-gemeente-van-inschrijving/overzicht.feature)
+  - [fout cases](./zoek-met-geslachtsnaam-voornamen-en-gemeente-van-inschrijving/fout-cases.feature)
+- zoek met postcode en huisnummer
+  - [overzicht](./zoek-met-postcode-en-huisnummer/overzicht.feature)
+  - [fout cases](./zoek-met-postcode-en-huisnummer/fout-cases.feature)
+- zoek met straat, huisnummer en gemeente van inschrijving
+  - [overzicht](./zoek-met-straatnaam-huisnummer-en-gemeente-van-inschrijving/overzicht.feature)
+  - [fout cases](./zoek-met-straatnaam-huisnummer-en-gemeente-van-inschrijving/fout-cases.feature)
+- zoek met nummeraanduiding identificatie
+  - [overzicht](./zoek-met-nummeraanduiding-identificatie/overzicht.feature)
+  - [fout cases](./zoek-met-nummeraanduiding-identificatie/fout-cases.feature)
+
+Het resultaat van deze operaties is een PersoonBeperkt lijst. Standaard bevat deze lijst alleen personen die in leven zijn. Om een overleden persoon te zoeken, moet de inclusiefOverledenPersonen parameter op true worden gezet. 
+
+Voor overleden personen wordt altijd het indicatieOverleden veld geleverd. Zie de [indicatie overleden](../overlijden-indicatie-overleden.feature) feature voor meer informatie over dit veld.
+
+
+## Raadplegen van personen
+
+Als de burgerservicenummer van de te bevragen personen wel bekend is, kan de volgende operatie worden gebruikt om gegevens van de persoon te raadplegen:
+
+- raadpleeg met burgerservicenummer
+  - [overzicht](./raadpleeg-met-burgerservicenummer/overzicht.feature)
+  - [fout cases](./raadpleeg-met-burgerservicenummer/fout-cases.feature)
+
+Het resultaat van deze operatie is een Persoon lijst.
```

Comment:
Ik vermoed dat sommige personen onder `Persoon lijst' wat anders verstaan dan wat er met de API wordt geleverd. Eigenlijk is het een gefilterde view op de 'Persoon lijst'.

Reply:
> Ik vermoed dat sommige personen onder `Persoon lijst' wat anders verstaan dan wat er met de API wordt geleverd. Is het niet eerder een gefilterde view op de 'Persoon lijst'?

Dit is een verwijzing naar de Persoon schema component. Ik heb het nu collectie/lijst genoemd